### PR TITLE
Update photo size

### PR DESCRIPTION
**Pull request by bwuzhang**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/bwuzhang/bwuzhang.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: bwuzhang 🐶
```
Comment (mitulsavani) [inline _data/bwuzhang/bwuzhang.md:2]: Please follow the given [instructions](https://github.com/education/graduation#3-add-your-profile-information) by removing the emoji to maintain the same format

Comment (bwuzhang) [reply _data/bwuzhang/bwuzhang.md:2]: is emoji not allowed? there's a flag emoji in on the institution line on the official template

Comment (mitulsavani) [reply _data/bwuzhang/bwuzhang.md:2]: While reviewing PR, I have not come across a single PR who has emoji on the name tag and I personally don't see a problem wiith it. Maybe you can try keeping it and see if it gets merged. ;)

Comment (Ahmeed2m) [reply _data/bwuzhang/bwuzhang.md:2]: According to this https://github.com/education/graduation/issues/348#issuecomment-631776551 There's no problems with emojis.

**Pull request merged**